vagrant_setup/go.sh

```
@@ -8,8 +8,9 @@ sudo echo 'export PATH=$PATH:/usr/local/go/bin' >> /etc/profile
 sudo echo 'export GOPATH=/usr/local' >> /etc/profile
 sudo echo 'export PATH=$PATH:$GOPATH/bin' >> /etc/profile
 sudo echo 'export GO111MODULE=on' >> /etc/profile
+sudo echo 'export CGO_ENABLE=0' >> /etc/profile
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use .. | sudo tee -a file

(SC2024)